src/test/java/uk/gov/companieshouse/service/dissolution/certificate/DissolutionCertificateLocationGeneratorTest.java

```diff
@@ -25,11 +27,13 @@
 
     @Test
     public void generateCertificateLocation_generatesAnS3UrlUsingThePdfBucketAndEnvName() {
+        final Dissolution dissolution = generateDissolution();
+
         when(dissolutionConfig.getDissolutionPdfBucket()).thenReturn("some-pdf-bucket");
         when(envConfig.getEnvironmentName()).thenReturn("some-env");
 
-        final String result = locationGenerator.generateCertificateLocation();
+        final String result = locationGenerator.generateCertificateLocation(dissolution);
 
-        assertEquals("s3://some-pdf-bucket/some-env/dissolution", result);
+        assertEquals("s3://some-pdf-bucket/some-env/dissolution/Apply-to-strike-off-and-dissolve-a-company-ABC123.pdf", result);
```

Comment:
I think you should manually set the reference in this test rather than relying on the fixture value

Reply:
Can do, though it's basically the same thing. If someone changed the fixtures, then this test would fail, showing it works ok

Reply:
tests should always define the data they assert upon. looking at this test in isolation, there's no way of knowing where the ABC123 came from.